CRM/Activity/Import/Form/MapField.php

```diff
@@ -71,6 +71,7 @@ public function preProcess() {
    */
   public function buildQuickForm() {
     // To save the current mappings.
+    $this->assign('loadedMapping', $this->get('savedMapping'));
```

Comment:
I see that `$this->get('savedMapping')` is called 3 times. Why not assign it to a variable once?
```suggestion
    $savedMapping = $this->get('savedMapping');
    $this->assign('loadedMapping', $savedMapping);
```

Reply:
@colemanw because it has no cost to speak of calling that  & the code can be moved around more easily when a function === a variable

Reply:
However, I'm gonna close this because it's actually not adequately fixing the issues - the tpl var is 'savedMapping' and I now think it should be removed